### PR TITLE
Approved content catalogue route text changes

### PR DESCRIPTION
**Pull request by Hendrikdj**
Route texts updated to all be around the same length, highlighting features of the route. Also added some download size values.

### CODE REVIEW

File: routes.json [hunk 1]
```diff
@@ -3,34 +3,36 @@
...
+        "description": "The historic Burrinjuck railway, built in 1907 and decommissioned in 1929, was a narrow gage route between Goondah and Burrinjuck in New South Wales, Australia. The line was used to transport workers, material and supplies during the construction of the Burrinjuck dam. Navigate the four German built 0-4-0 Krauss steam locomotives that transported approximately 100 tonnes of material per day, along this 26 mile / 42 km long route with challenging steep gradients.",
```
Comment (cjakeman) [inline routes.json:6]:
```suggestion
        "description": "The historic Burrinjuck railway, built in 1907 and decommissioned in 1929, was a narrow gauge route between Goondah and Burrinjuck in New South Wales, Australia. The line was used to transport workers, material and supplies during the construction of the Burrinjuck dam. Navigate the four German built 0-4-0 Krauss steam locomotives that transported approximately 100 tonnes of material per day, along this 26 mile / 42 km long route with challenging steep gradients.",
```

**Review by cjakeman — COMMENTED** [review]
Hi Henk,

Many thanks for all this work. Just what we needed.

The content is very good - thanks.

I found a single typo but also 2 unwanted files "routes.json~" and ".DS_Store" which I guess your editing tools created.
Would you be able to delete these, please?

Thanks,

Chris

**Comment by Hendrikdj** [comment]
Deleted Mac garbage files

**Comment by Hendrikdj** [comment]
Hi Chris, I removed the files and reopened the pull request. Both are mac "garbage". The routes.json~ is a backup before edit changes and the DS_store file seems to be a file that the mac uses to store directory attributes. It is a hidden file on the mac but comes over with the pull request. I will keep that in mind for next time.I think the "gauge" typo you corrected is still in the pull request.Thanks!Henk 

    On Friday, April 5, 2024 at 12:38:58 PM MDT, Chris Jakeman ***@***.***> wrote:  
 
 


@cjakeman commented on this pull request.

Hi Henk,

Many thanks for all this work. Just what we needed.

The content is very good - thanks.

I found a single typo but also 2 unwanted files "routes.json~" and ".DS_Store" which I guess your editing tools created.
Would you be able to delete these, please?

Thanks,

Chris

**Review by cjakeman — APPROVED** [review]
Hi Henk,

Approved and merged with thanks,

Chris

**Comment by cjakeman** [comment]
Merged

**Pull request merged**